### PR TITLE
WASM Travis builds

### PR DESCRIPTION
**Pull request by motin**
Leads to builds that at least goes until 93% compilation completion, as demonstrated by https://travis-ci.com/github/motin/bergamot-translator/jobs/484229989

### CODE REVIEW

File: build-wasm.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+# Usage: ./build-wasm.sh
+
+set -e
+set -x
+
+# run script from the context of the script-containing directory
+cd "$(dirname $0)"
+
+# This file replicates the instructions found in ../bergamot-translator/README.md under "Build WASM"
```
Comment (abhi-agg) [inline build-wasm.sh:11]: Shouldn't it be `bergamot-translator/README.md` instead of `../bergamot-translator/README.md`?

File: build-wasm.sh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+# 4. Compile
+#     1. Create a folder where you want to build all the artefacts (`build-wasm` in this case)
+if [ ! -d "build-wasm" ]; then
+  mkdir build-wasm
+fi
+cd build-wasm
+
+#     2. Compile the artefacts
+emcmake cmake -DCOMPILE_WASM=on -DPACKAGE_DIR="$(pwd)/../models/" ../
```
Comment (abhi-agg) [inline build-wasm.sh:65]: Nit pick. `$(pwd)/../models/` is not required. Just `../models/` will also work.

**Comment by abhi-agg** [comment]
If everything builds on Travis, I would be happy to +1 to it :)

**Comment by motin** [comment]
Builds currently fails with:
```
[ 93%] Built target ssplit

No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.

Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received

The build has been terminated
```

**Comment by abhi-agg** [comment]
Can we start the build again? I can't even see any error.

**Comment by motin** [comment]
> Can we start the build again? I can't even see any error.

The error is at the bottom of https://travis-ci.com/github/motin/bergamot-translator/jobs/484229989, and unfortunately trying to restart the job ends up with "Oh no! An error occurred. The build could not be restarted."

**Comment by motin** [comment]
We will use CircleCI instead, closing this and will be back with a corresponding one for Circle.

**Pull request closed without merging**